src/views/forms/TradeForm/TradeFormInputs.tsx

```diff
@@ -55,14 +56,15 @@ export const TradeFormInputs = () => {
 
   const midMarketPrice = useAppSelector(getCurrentMarketMidMarketPrice, shallowEqual);
   const [hasUserChangedLimitInput, setHasUserChangedLimitInput] = useState(false);
+  const activeDialog = useAppSelector(getActiveDialog);
 
   useEffect(() => {
     setHasUserChangedLimitInput(false);
   }, [marketId, type?.rawValue]);
 
   useEffect(() => {
     // when limit price input is empty and mid price is available, set limit price input to mid price
-    if (!midMarketPrice || !needsLimitPrice || hasUserChangedLimitInput) {
+    if (activeDialog != null || !midMarketPrice || !needsLimitPrice || hasUserChangedLimitInput) {
```

Comment:
hmmmm. The activeDialog check would work but is still kind of suspect.

SLTP - currently in a dialog
TransferMargin - currently in a dialog
Withdraw/Deposit - currently in a dialog

If they were ever used outside of dialogs we would have issues.

Reply:
agreed, was just trying to salvage as much functionality as I can